index.html

```diff
@@ -491,37 +492,26 @@ <h4 id="editor-linux">Linux</h4>
   <div class="row">
     <div class="col-md-4">
       <h4 id="python-windows">Windows</h4>
-      <ul>
-        <li>
-          Download and
-          install <a href="https://store.continuum.io/cshop/anaconda/">Anaconda</a>.
-        </li>
-        <li>
-          Download the default Python 2 installer (do not follow the link to version 3).
-          Use all of the defaults for installation
-          <em>except</em> make sure to check
-          <strong>Make Anaconda the default Python</strong>.
-        </li>
-      </ul>
+      <p>
+        Download <a href="http://continuum.io/downloads#py34">this Python 3 installer</a>
```

Comment:
Ouch.  That's really bad news :-(  Have you spoken to Continuum about 
fixing it?